poly/constants.go

```diff
@@ -0,0 +1,15 @@
+package poly
+
+import (
+	"github.com/dedis/crypto/edwards"
+)
+
+const (
+	MINIMUM = iota
+	MODERATE
+	MAXIMUM
+)
+
+var SECURITY int = MAXIMUM
+
+var SUITE = edwards.NewAES128SHA256Ed25519(true)
```

Comment:
This *definitely* needs to be fixed before merge.  It's OK to have a global var declaring a particular suite to be used for testing purposes, but that should be only in a _test.go file, and I would call it 'testSuite' or just lowercase 'suite' to be consistent with Go conventions and other package.  And it's definitely *not* OK to have the non-test code dependent on a global SUITE variable like this: all code that depends on the ciphersuite needs to be parameterized properly.